### PR TITLE
Cover message parsing via `.fromCell` and `.fromSlice` with tests

### LINKED ISSUE #453: Message parsing via `.fromCell` and `.fromSlice` is not tested
Messages are just structs under the hood, so they are already supported for these operations and we need to test them.
Also messages support isn't mentioned in changelog too.

### PR DESCRIPTION
**Pull request by Gusarich**
Closes #453 

- [x] I have updated CHANGELOG.md
- [x] I have added tests to demonstrate the contribution is correctly implemented: this usually includes both positive and negative tests, showing the happy path(s) and featuring intentionally broken cases
- [x] I have run all the tests locally and no test failure was reported
- [x] I have run the linter, formatter and spellchecker
- [x] I did not do unrelated and/or undiscussed refactorings

### CODE REVIEW

**Review by novusnota — COMMENTED** [review]
Could you please add more types to tests, like String, Bool and, say, StringBuilder (that one is rarely used as a Struct field, but just in case)?

**Comment by anton-trunov** [comment]
Sorry, there are merge conflicts from #445 now

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `loadBool` method for `Slice` type: PR [#412](https://github.com/tact-lang/tact/pull/412)
 - CLI flag `--with-decompilation` to turn on decompilation of BoC files at the end of the compilation pipeline: PR [#417](https://github.com/tact-lang/tact/pull/417)
 - Support more Tact expressions in the constant evaluator: condition expressions, struct instances, struct field access, `emptyMap()`: PR [#432](https://github.com/tact-lang/tact/pull/432) and PR [#445](https://github.com/tact-lang/tact/pull/445)
-- The `fromCell` and `fromSlice` methods for struct parsing: PR [#418](https://github.com/tact-lang/tact/pull/418)
+- Support more Tact expressions in the constant evaluator: condition expressions, struct instances, struct field access, `emptyMap()`: PR [#432](https://github.com/tact-lang/tact/pull/432)
```
Comment (anton-trunov) [inline CHANGELOG.md]: looks like something went wrong during the merge

**Comment by Gusarich** [comment]
@anton-trunov done

**Comment by anton-trunov** [comment]
> Could you please add more types to tests, like String, Bool and, say, StringBuilder (that one is rarely used as a Struct field, but just in case)?

Let's open a new issue on this, maybe with even a broader scope? If there are any bugs in this, we'll release v1.4.1 :)

**Review by anton-trunov — APPROVED** [review]

**Pull request merged**